### DISCUSSION TITLE
Installation of 2.4.90 on ProLiant DL360 Gen10 failing

### DISCUSSION THREAD

**Discussion by ejgh-oe** [Security-Onion-Solutions/securityonion#13551] [General] [answered]
Hi,
I've got a ProLiant DL360 Gen10 equipped with > 512GB RAM and around 7TB disk space that I want to use as sensor node. So I started out with the 2.4.90 install iso. 
After asking whether it's OK to overwrite everything...

![image](https://github.com/user-attachments/assets/c2da5937-35ac-432c-9418-6bb88d4c0b91)

...the installer went on and crashed when trying to access disks:

![image](https://github.com/user-attachments/assets/618cd729-1092-49f3-be15-9fb4763ac612)

Wondering that that `/dev/sdd` mentioned in the above message is, I went to the shell and did an `fdisk` for `/dev/sda`, `/dev/sdb` and `/dev/sdd`

Here's what I got:

![image](https://github.com/user-attachments/assets/bde10c9c-749a-4cdd-8186-274190662ac2)

According to that `/dev/sdd` is a flash storage with only 320KB in size (!)

Looking at the log in /tmp/anaconda-tb* looking for further details I found something interesting: doing a `grep /dev/sda /tmp/anaconda-db*` here's what I got

![image](https://github.com/user-attachments/assets/5f023946-b40c-4ad9-887f-ce081f6e6419)

when doing a  `grep /dev/sdd /tmp/anaconda-db*` things look quite different:

![image](https://github.com/user-attachments/assets/46179eb1-34aa-4712-8341-1b167dec1d60)

In order to get an idea as to what this 320KB device is, I started the Ubuntu installer and when the installer came to disk partitioning, the 320KB device showed up as "HPE_SPI_FLASH"

![image](https://github.com/user-attachments/assets/9cf81680-3c52-4b2f-9b2c-ad111f7df680)

So there obviously is a small flash disk in the server that by simple presence makes the SO-installer crash.

So here are my questions:

- Shouldn't the SO-installer ask which disks SO is to be installed upon (at least this is what I know from my previous installations)?
- Any way to prevent the installer from probing (and touching) all sorts of disk devices upon installation?

Thanks much in advance for your help.

**Comment by cm-ops** [comment]
The installer should prompt you for which drive you want to install the OS, whether you want to set `/nsm` on the same disk, if not, what disk to set `/nsm`.

**Reply by ejgh-oe** [reply]
> The installer should prompt you for which drive you want to install the OS, whether you want to set /nsm on the same disk, if not, what disk to set /nsm.

Hi,
That's exactly what I would have expected from all previous installations: installer telling you "I'll wipe everything", asking for userid/pw for the admin-user and finally providing a list of disk devices asking where the OS and /nsm should go.

What happens in this case though is "I'll wipe everything", then asking for userid/pw - and "boom". To me it seems like the installer doesn't even make it to the point where it should ask about the installation targets (disks).

Would it help if I try to get hold of the log in /tmp? (This would mean my trying to get the network interface up&running despite the installer crashed)

**Reply by reyesj2** [reply]
Are there multiple non-removable storage drives in this system? If there is only one storage drive the installer would use that, otherwise prompt for which disk to use.

**Reply by ejgh-oe** [reply]
> Are there multiple non-removable storage drives in this system? If there is only one storage drive the installer would use that, otherwise prompt for which disk to use.

Yes, from the screenshot above taken from the Ubuntu install there are other non-removable media in the server (red border in the screenshot above), amongst others the 320KB device that seems to crash the SO installer

Also tried with Debian - again the installer sees the exact same 320KB device

![image](https://github.com/user-attachments/assets/61f73125-5958-4dd4-8625-77e93dc33520)

What I could do in the meantime is to remove the GB micro SD card and see if this solves the problem - I'll keep you posted ...

**Reply by ejgh-oe** [reply, marked as answer]
Today we pulled the builtin microSD card from the server and guess what: all problems are gone :-)
With only one disk (Raid1) available the installer runs straight through without any hiccups, so we can close this as "solved" 👍
